examples/kwil/copy_investing.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+"""
+This copies data into kwil
+"""
+
+import logging
+import hashlib
+from functools import partial
+from icecream import ic
+
+from dotenv import load_dotenv
+import pandas as pd
+
+from typing import Dict
+import hashlib
+
+from truflation.data.pipeline_details import PipeLineDetails
+from truflation.data.source_details import SourceDetails
+from truflation.data.export_details import ExportDetails
+from truflation.data.connector import get_database_handle
+from truflation.data.connector import connector_factory
+from truflation.data.connectors.kwil import ConnectorKwil
+
+load_dotenv()
+logger = logging.getLogger(__name__)
+#ic.disable()
+
+# Name
+PIPELINE_NAME = "Generate USD"
+
+
+ROUNDING = 6
+TABLES = [
+    'com_investing_dutch_ttf_natural_gas_futures',
+    'com_investing_usd_eur_historical'
+]
```

Comment:
How does these tables relate to the USD/CNY?

Reply:
@j-dabrowski

Reply:
They don't, am correcting this
Is the goal to collect multiple varieties of lithium, eg. lithium 99.9, 99, 60?

https://www.investing.com/commodities/99.9-min-china-lithium-metal-futures-historical-data
https://www.investing.com/commodities/lithium-carbonate-99-min-china-futures-historical-data
https://www.investing.com/commodities/lithium-cobalt-oxide-60-min-china-futures-historical-data

Reply:
todo: 
find the endpoint to return prices for each of these commodities
have pipeline details source the data using these endpoints

Reply:
@zolotokrylin

Reply:
@joequant